### ISSUE TITLE
Unable to run ngl i on alpha.13

### ISSUE THREAD

**Issue by besuhoff** [gonzofish/angular-library-set#34]
```bash
➜  academy-ui ./node_modules/.bin/ngl i  
/usr/bin/env: «node\r»: No such file or directory
```
At the same time I can correctly run
```bash
➜  academy-ui env node   
> 
```
Looks like it contains incorrect new line symbol
OS: `Ubuntu 16.10`

**Comment by gonzofish** [comment]
Thanks for bringing this up. I'll look into it.

**Comment by gonzofish** [comment]
Fixed with changes in fe01e01

Also republished to alpha.14 in NPM

**Comment by gonzofish** [comment]
I tested this on a CentOS VM and it worked. Please, if you could @besuhoff, let me know if it works for you
